keyboards/ok60/config.h

```diff
@@ -21,7 +21,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4b36 // K6 - ok60
```

Comment:
It's preferable for the value to be fully uppercase.

```suggestion
#define VENDOR_ID       0x4B36 // K6 - ok60
```